### ISSUE TITLE
Add `include_preprocessing` argument in EfficientNet models. 

### ISSUE THREAD

**Issue by innat** [keras-team/keras#16081]
**System information**.

TensorFlow version (you are using): 2.8
Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state**.

I think most of the keras application doesn't have the built-in input pre-processing layers, and that's why it's often troubling to config a data pipeline to match up the current efficient-net (v1) models. Let's say, in your project, in the data pipeline, you normalize the data and use dense-net for the model. Now, you switch to efficient-net (v1) and for this, you need to go back to the data pipeline and remove all sorts of normalization on input. 

For a small-scale experiment, it doesn't look too much but when you start experimenting with lots of different models (+ models that are not officially provided), this becomes problematic to change the data pipeline over and over again. 

FYI, [EfficientNet V2](https://github.com/keras-team/keras/blob/v2.8.0/keras/applications/efficientnet_v2.py#L920-L932) and also MobileNet V2 have the `include_preprocessing` argument. 

**Will this change the current api? How?** Yes.

from 

```python
tf.keras.applications.efficientnet.EfficientNetB0(
    include_top=True, weights='imagenet', input_tensor=None,
    input_shape=None, pooling=None, classes=1000,
    classifier_activation='softmax', **kwargs
)
```

to 

```python
tf.keras.applications.efficientnet.EfficientNetB0(
    include_top=True, weights='imagenet', input_tensor=None,
    input_shape=None, pooling=None, classes=1000,
    classifier_activation='softmax', 
     include_preprocessing=True  # < --------------
)
```

**Who will benefit from this feature?**
ML practitioners who use keras. 

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**

- Do you want to contribute a PR? (yes/no): No. 
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing):

**Comment by innat** [comment]
@jvishnuvardhan any update? I believe it's a small fix.

**Comment by mattdangerw** [comment]
Yep this looks like a small fix, can just add the argument, default it to true, and use it to gate the Rescaling and Normalization layers [here](https://github.com/keras-team/keras/blob/v2.8.0/keras/applications/efficientnet.py#L321). EfficientnetV2 is a good example.

Marking this as open for contributions, as I am not sure anyone on the team will have immediate bandwidth to pick this up. Thanks for filing!

**Comment by old-school-kid** [comment]
@innat @mattdangerw 
I would like to work on this. 
I went through the [EfficientNet V2 code](https://github.com/keras-team/keras/blob/master/keras/applications/efficientnet_v2.py#L932) and it seems whether or not `include_preprocessing` is True the image is being dived by 128 with a offset of -1. So wont that raise the same issue again?

**Comment by fperezgamonal** [comment]
Hello all,

I'm currently fine-tuning an EfficientNetV2S network and happened to take quite a deep dive into the code after finding the already solved (to master) issue with dropout layers #16144 (to ensure there weren't other bugs).
I also use my own normalisation code and hence set `include_preprocessing = True`. @old-school-kid , if you take a closer look at [the EfficientNetV2 code,](https://github.com/keras-team/keras/blob/c9e2b4dbf1a51f721c9364116ab4d8ccca8d9e44/keras/applications/efficientnet_v2.py#L920) both normalisations (`if else` clause) are inside the same `if` statement so if `include_preprocessing = False`, neither normalisation will be applied.

It is clearer if you look at the tabulation of the `if-else` block vs the external if and the following code:+1: 
![image](https://user-images.githubusercontent.com/19503950/161083660-b0ebbfea-12ec-476b-bc01-b8190879371c.png)
